docs/how_to/use_optimizationlevel.rst

```diff
@@ -0,0 +1,333 @@
+#####################################
+Setting transpiler optimization level
+#####################################
+
+This guide shows you how to use the :attr:`~qiskit.transpile.optimization_level` 
+parameter with :meth:`~qiskit.compiler.transpile`.
+
+:attr:`~qiskit.transpile.optimization_level` helps you to optimize your quantum circuit.
+This parameter takes an integer which can be a value between 0 and 3,
+where the higher the number, the more optimized the result.
+You can find more information about this parameter
+`here <https://qiskit.org/documentation/tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.html#Preset-Pass-Managers>`_.
+
+Initialize the quantum circuit
+==============================
+
+The effect of setting the :attr:`~qiskit.transpile.optimization_level` will differ depending on the backend you are using.
+For this example, you will explore the `CSWAP <https://qiskit.org/documentation/stubs/qiskit.circuit.QuantumCircuit.cswap.html>`_ gate,
+which is a three qubit gate which will be transpiled into one and two qubit gates.
+
+.. testcode::
+
+    from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister 
+    from qiskit.compiler import transpile
+    from qiskit.providers.fake_provider import FakeQuito
+
+    backend = FakeQuito()
+
+    qc = QuantumCircuit(3) # Initialize the quantum circuit with 3 qubits.
+    
+    qc.cswap(qr[0],qr[1],qr[2]) # Add the cswap gate to the quantum circuit.
```

Comment:
The variable `qr` is not defined anymore 🙂 Could you run all code cells to ensure they execute without error?
```suggestion
    qc.cswap(0, 1, 2) # Add the cswap gate to the quantum circuit.
```